src/ldes/LDESinLDP.ts

```diff
@@ -341,4 +341,36 @@ export class LDESinLDP implements ILDES {
             }
         }
     }
+
+
+    /**
+     * Returns the last members added to the container based on the timestamp.
+     * @returns {Promise<Quad[]>}
+     */
+    public async getLastMembers(): Promise<Quad[]> {
+        const metadata = await this.extractLdesMetadata();
+        const relations = filterRelation(metadata, new Date(0), new Date())
```

Comment:
```suggestion
        const relations = filterRelation(metadata, new Date(), new Date())
```
Normally this is going to give the most recent fragment, which means the rest of the code will work just as is